deployment/readability/templates/10-readability-deployment.yaml

```
@@ -0,0 +1,86 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: arxiv-readability
+  labels:
+    subsystem: "{{ .Values.labels.subsystem }}"
+    container: readability
+    service-group: ui
+    log-style: uwsgi
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        subsystem: "{{ .Values.labels.subsystem }}"
+        container: readability
+        service-group: ui
+        log-style: uwsgi
+      annotations:
+        prometheus.io/scrape: 'true'
+    spec:
+      containers:
+      - name: arxiv-readability
+        image: arxiv/readability:{{ .Values.imageTag }}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8000
+        env:
+        - name: DATABASE_URL
+          value: "{{ .Values.database_url }}"
+        - name: CELERY_BROKER_URL
+          value: "redis://{{ .Values.redis.host }}:{{ .Values.redis.port}}"
+        - name: CELERY_RESULT_BACKEND
+          value: "redis://{{ .Values.redis.host }}:{{ .Values.redis.port}}"
+        - name: SECRET_KEY
+          value: "{{ .Values.secret_key }}"
+        - name: SCRIPT_NAME
+          value: "{{ .Values.script_name }}"
+        - name: FORCE_SCRIPT_NAME
+          value: "{{ .Values.script_name }}"
+        - name: ALLOWED_HOSTS
+          value: "*"
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: arxiv-readability-worker
+  labels:
+    subsystem: "{{ .Values.labels.subsystem }}"
+    container: readability-worker
+    service-group: backend
+    log-style: celery
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        subsystem: "{{ .Values.labels.subsystem }}"
+        container: readability-worker
+        service-group: backend
+        log-style: celery
+      annotations:
+        prometheus.io/scrape: 'true'
+    spec:
+      containers:
+      - name: arxiv-readability-worker
+        image: arxiv/readability:{{ .Values.imageTag }}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 8000
+        command:
+          - "sh"
+          - "-c"
+          - >
+            watchmedo auto-restart --recursive -d . -p '*.py' -- celery -A arxiv_html worker -l info
```

```suggestion
            celery -A arxiv_html worker -l info
```

`watchmedo` is just for restarting when code changes in development.